src/Ng/Ng.csproj

```diff
@@ -178,23 +175,20 @@
       <Version>5.0.0-preview1.5665</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Services.Configuration">
-      <Version>2.2.2</Version>
+      <Version>2.57.0</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Services.Logging">
-      <Version>2.25.0-master-30088</Version>
+      <Version>2.57.0</Version>
```

Comment:
Woah, big jump! Maybe we should deploy the jobs and ensure everything is working okay in DEV.

Reply:
**All** jobs need redeploying to DEV and verification of heartbeats :) Need to come up with a deployment plan for these.

Reply:
Also, I think we need to do a better job of keeping our dependencies up to date throughout our repositories to avoid these big jumps. Regular updates FTW.